dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoPresenter.java

```diff
@@ -241,10 +242,11 @@ protected Object call() {
                     refresh(buttonCell.getTreeTableView(), jobInterruptionStore, endpointInfo);
                 }));
                 recoverInterruptedJob.setOnFailed(SafeHandler.logHandle(recoverInterruptedJobFailedEvent -> {
+                    LOG.error("Failed to recover", recoverInterruptedJob.getException());
                     loggingService.logMessage("Failed to recover " + filesAndFolderMap.getType() + " job " + endpointInfo.getEndpoint(), LogType.ERROR);
                     refresh(buttonCell.getTreeTableView(), jobInterruptionStore, endpointInfo);
                 }));
-                recoverInterruptedJob.setOnCancelled(SafeHandler.logHandle(recoverInterruptedJobCancelledEvent -> {
+                recoverInterruptedJob.setOnCancelled(SafeHandler.logHandle((WorkerStateEvent recoverInterruptedJobCancelledEvent) -> {
```

Comment:
Is this cast really necessary?

Reply:
For some reason yes, we can't resolve getException without it